### PR TITLE
feat(price-feeder): hardcoded accept quote list

### PR DESCRIPTION
**Pull request by adamewozniak**
## Description

Adds a hardcoded acceptable quote list to the price feeder.

Addresses the first part of #1117

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: price-feeder/config/config.go [hunk 1]
```diff
@@ -52,6 +52,16 @@ var (
...
 	maxDeviationThreshold = sdk.MustNewDecFromStr("3.0")
+
+	// SupportedQuotes defines a lookup table for which assets we support
+	// using as quotes.
+	SupportedQuotes = map[string]struct{}{
+		DenomUSD: {},
+		"USDC":   {},
+		"USDT":   {},
+		"BTC":    {},
+		"ETH":    {},
```
Comment (RafilxTenfen) [inline price-feeder/config/config.go:63]: should Atom be here as well?

Comment (RafilxTenfen) [reply price-feeder/config/config.go:63]: DAI also

**Review by RafilxTenfen — APPROVED** [review]
Maybe add Atom and DAI ?

File: price-feeder/config/config.go (hunk 1 above)
Comment (adamewozniak) [reply price-feeder/config/config.go:63]: DAI yes, atom we're not adding until we can measure volume ( ref #1117 )

Comment (adamewozniak) [reply price-feeder/config/config.go:64]: c332a4b71c9d7318e052723cc637cfc1a0d98641

File: price-feeder/config/config.go [hunk 2]
```diff
@@ -52,6 +52,17 @@ var (
...
 	// deviations which validators are able to set for a given asset.
 	maxDeviationThreshold = sdk.MustNewDecFromStr("3.0")
+
+	// SupportedQuotes defines a lookup table for which assets we support
+	// using as quotes.
+	SupportedQuotes = map[string]struct{}{
+		DenomUSD: {},
+		"USDC":   {},
+		"USDT":   {},
+		"DAI":    {},
```
Comment (robert-zaremba) [inline price-feeder/config/config.go:62]: Any motivation why we need 3 USD denoms, rather than 1 or 2?

**Pull request merged**